### PR TITLE
Add ocp-4.4 jobs

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml [hunk 1]
```diff
@@ -480,6 +480,94 @@ presubmits:
...
+            - "/usr/local/bin/runner.sh"
+            - "/bin/bash"
+            - "-c"
+            - "cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io && KUBEVIRT_PROVIDER=ocp-4.4 make cluster-up"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: "15Gi"
```
Comment (phoracek) [inline github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml]: Is it enough to set it up?

Comment (qinqon) [reply github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml]: Well it was for ocp-4.3, we cannot know if we don't merge this.

Comment (phoracek) [reply github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml]: ack

**Pull request merged**